### PR TITLE
Add optional pytest plugin to prysk project

### PR DESCRIPTION
**Pull request by Nicoretti**

### CODE REVIEW

File: docs/pytest-plugin.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Pytest-Plugin
+==============
+Prysk optionally can be installed with pytest support. If the user chooses to do so,
+various features an benefits provided by pytest and its plugins will be also available
```
Comment (kynan) [inline docs/pytest-plugin.rst:4]: typo: an -> and

File: pytest_prysk/__init__.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
+try:
+    import pytest
+except ImportError as ex:
+    error_msg = (
+        "Pytest not available, "
+        "make sure you install pryskk using the extra [].\n"
```
Comment (kynan) [inline pytest_prysk/__init__.py:6]: Something missing? "using the extra [pytest-prysk]" ?

File: pytest_prysk/__init__.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+class File(pytest.File):
+    def collect(self) -> Iterable[Union[PytestItem, Collector]]:
+        yield Item.from_parent(self, options=self.config.option, name=self.path.name)
+
+
+class Item(pytest.Item):
+    def __init__(self, options, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.add_marker("prysk")
+        self.tmpdir = TemporaryDirectory()
```
Comment (kynan) [inline pytest_prysk/__init__.py:124]: Since you're only using this in `runtest`, why not use it as a context manager in there only?

**Comment by Nicoretti** [comment]
@kynan thanks for the code-review, appreciate it!
Addressed your feedback in #164.

File: pytest_prysk/__init__.py (hunk 3 above)
Comment (Nicoretti) [reply pytest_prysk/__init__.py:124]: Good point.

File: pytest_prysk/__init__.py (hunk 2 above)
Comment (Nicoretti) [reply pytest_prysk/__init__.py:6]: 👍

File: docs/pytest-plugin.rst (hunk 1 above)
Comment (Nicoretti) [reply docs/pytest-plugin.rst:4]: Thanks!

**Pull request merged**